cluster-operator/src/test/java/io/strimzi/operator/cluster/model/cruisecontrol/CapacityConfigurationTest.java

```diff
@@ -0,0 +1,525 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model.cruisecontrol;
+
+import io.fabric8.kubernetes.api.model.Quantity;
+import io.fabric8.kubernetes.api.model.ResourceRequirements;
+import io.fabric8.kubernetes.api.model.ResourceRequirementsBuilder;
+import io.strimzi.api.kafka.model.kafka.JbodStorageBuilder;
+import io.strimzi.api.kafka.model.kafka.KafkaSpec;
+import io.strimzi.api.kafka.model.kafka.KafkaSpecBuilder;
+import io.strimzi.api.kafka.model.kafka.PersistentClaimStorageBuilder;
+import io.strimzi.api.kafka.model.kafka.Storage;
+import io.strimzi.api.kafka.model.kafka.cruisecontrol.BrokerCapacity;
+import io.strimzi.api.kafka.model.kafka.cruisecontrol.BrokerCapacityBuilder;
+import io.strimzi.operator.cluster.KafkaVersionTestUtils;
+import io.strimzi.operator.cluster.model.KafkaVersion;
+import io.strimzi.operator.cluster.model.MockSharedEnvironmentProvider;
+import io.strimzi.operator.cluster.model.NodeRef;
+import io.strimzi.operator.cluster.model.SharedEnvironmentProvider;
+import io.strimzi.operator.common.Reconciliation;
+import io.strimzi.test.annotations.ParallelTest;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CapacityConfigurationTest {
+    private static final SharedEnvironmentProvider SHARED_ENV_PROVIDER = new MockSharedEnvironmentProvider();
+    private static final KafkaVersion.Lookup VERSIONS = KafkaVersionTestUtils.getKafkaVersionLookup();
+    private static final Map<String, ResourceRequirements> RESOURCES = createResources("400m", "0.5");
+    private static final KafkaSpec KAFKA_SPEC = new KafkaSpecBuilder()
+            .withNewKafka()
+            .endKafka()
+            .withNewCruiseControl()
+            .endCruiseControl()
+            .build();
+    private static final Set<NodeRef> NODES = Set.of(
+            new NodeRef("my-cluster-brokers-0", 0, "brokers", false, true),
+            new NodeRef("my-cluster-brokers-1", 1, "brokers", false, true),
+            new NodeRef("my-cluster-brokers-2", 2, "brokers", false, true)
+    );
+
+    @ParallelTest
+    public void testBrokerCapacity() {
+        KafkaSpec kafkaSpec = new KafkaSpecBuilder(KAFKA_SPEC)
+            .withNewCruiseControl()
+                .withNewBrokerCapacity()
+                    .withCpu("2575m")
+                    .withInboundNetwork("50000KB/s")
+                    .withOutboundNetwork("50000KB/s")
+                    .endBrokerCapacity()
```

Comment:
Wrong indentation. It should be more on the left :-)

Reply:
Fixed.